pygin/pygin/src/panel.cpp

```diff
@@ -3,6 +3,27 @@
 #include "panel.hpp"
 #include "far_api.hpp"
 #include "py.integer.hpp"
+#include "python.hpp"
+
+#include <ranges>
+
+namespace {
+
+void WINAPI free_py(void* UserData, const struct FarPanelItemFreeInfo* Info)
+{
+	Py_XDECREF(UserData);
```

Comment:
I'd prefer keeping low level python stuff in py.object.cpp.
Maybe build a temporary `object` and let its dtor do the thing?

Reply:
Fixed in 25bec961a4332cba0ae5649d2dd13af0bff6261c.